src/foam/nanos/boot/NSpec.js

```diff
@@ -173,7 +173,8 @@ foam.CLASS({
 
         Interpreter shell = new Interpreter();
         try {
-          shell.set("x", x);
+          foam.core.X y = x.put("logger", new foam.nanos.logger.StdoutLogger());
```

Comment:
Perhaps in the root context that we build in Boot.java, we can put a StdoutLogger as "logger" in the context.  Then any logging that happens before the real logger service is loaded will just use that logger.

Reply:
We've got another branch/PR were we are implementing the real solution for this, which involves two Boot.java service setups, the first dealing with logging, and once complete, the regular services setup.  We are going to leave this PR and that for NotificationLogging until it's complete.